docs/policy/repo-layout-migration-2025.md

```diff
@@ -0,0 +1,45 @@
+OSG Software Repo Layout Migration
+==================================
+
+11 December 2024
+
+On Monday, January 6th, 2025, the OSG Software Team will be upgrading the server that hosts the OSG Software repos
+(<https://repo.osg-htc.org> a.k.a. <https://repo.opensciencegrid.org>),
+which will result in changes to the directory layout for the OSG 23 and OSG 24 YUM repos.
+
+Users should not be affected since the URLs of the repos themselves will not change.
+However, mirrors will have to take some action to avoid rsyncing those repos from scratch.
+
+Perform the following steps:
+
+1.  Turn off rsync from `repo-rsync.opensciencegrid.org` (a.k.a. `repo-rsync.osg-htc.org`)
+
+2.  Download the migration script from <https://github.com/osg-htc/osg-repo-scripts/blob/el9/migrate.py>
+    ([direct link](https://raw.githubusercontent.com/osg-htc/osg-repo-scripts/refs/heads/el9/migrate.py))
```

Comment:
Do we want to provide a checksum or something?

Reply:
I don't think anybody will verify them on a file this small.